### PR TITLE
Update REPLICATION.md

### PR DESCRIPTION
**Pull request by michaeldarisse**
Include instructions for running `scan_packages.do`.

### CODE REVIEW

**Comment by larsvilhuber** [comment]
@michaeldarisse Do not have them change the installed packages. Rather, the program spits out an Excel file. In that Excel file, you can mark whether a package was correctly identified (because the author has already mentioned it). 

Also, the purpose of running the program is to give replicators a heads-up about packages that might be necessary to install, when the author has NOT mentioned which ones are needed. They would then subsequently know to add those packages themselves to the `config.do`.

File: REPLICATION.md [hunk 1]
```diff
@@ -167,6 +167,8 @@ Stated Requirements
 Missing Requirements
 --------------------
 
+> INSTRUCTIONS: If the replication package contains Stata programs and restricted-access data that the Lab was unable to access, run `tools/Stata_scan_code/scan_packages.do`. To do so, ensure that you update the global `codedir` and add any required packages mentioned in the README to the local `ssc_packages` on line 69. Review the output file - the program may pick up some false positives (packages that the script thinks are missing but are actually not Stata commands at all), use your judgement and include plausibly missing packages to the missing requirements list below.
```
Comment (larsvilhuber) [inline REPLICATION.md:170]: Should only update the global `codedir`, not the `ssc_packages`.

The output should then be copy-pasted into the "Missing requirements", coming from the Excel spreadsheet.

Comment (michaeldarisse) [reply REPLICATION.md]: @larsvilhuber Revised.

**Review by larsvilhuber — APPROVED** [review]
Looks good.

**Comment by larsvilhuber** [comment]
However, the only trouble is that this is "replication template". All changes to this are lost. you have to make changes to https://github.com/AEADataEditor/replication-template-development

**Pull request closed without merging**